akka-http-core/src/main/scala/akka/http/impl/engine/parsing/HttpMessageParser.scala

```diff
@@ -143,6 +146,7 @@ private[http] trait HttpMessageParser[Output >: MessageOutput <: ParserOutput] {
         case EmptyHeader =>
           val close = HttpMessage.connectionCloseExpected(protocol, ch)
           setCompletionHandling(CompletionIsEntityStreamError)
+          debugMessage(s"calling parseEntity with contentLength=$clh isChunked=$isChunked")
```

Comment:
This will allow comparing content length (if present) with length of data received (log lines in `emit()` below).